apps/crn-frontend/src/shared-research/api.ts

```diff
@@ -145,6 +145,32 @@ export const getDraftResearchOutputs = async (
   };
 };
 
+// As algolia has reduced data we need to fetch data from cms for `Export as CSV` feature
+export const getResearchOutputsFromCMS = async (
```

Comment:
The export using source filters (`teams`/`working groups`) is not working properly
<img width="1440" alt="Screenshot 2023-11-29 at 11 11 05" src="https://github.com/yldio/asap-hub/assets/16595804/8865aef3-94e0-4b61-a3d0-0af6da9346bc">